COMMUNITY_LADDER.md

```diff
@@ -162,10 +169,12 @@ An **Approver** is a Member who takes ownership of a specific part of the projec
     * You can recommend and review other Members for the Approver role.
 
 * **How to Become an Approver**:
-    1. Submit a PR to the team or member management file for your specific review area.
-    2. Get approval from at least two existing Maintainers.
 
-We have automated tools that may assign PRs to Approvers based on their designated areas of responsibility.
+  Based on the Member selection process above, add additional requirements:
+
+  1. Get nominated by an existing Approver or Maintainer.
+  2. Must document your contribution history and explain why you're suitable for this role, while listing the specific areas you want to be responsible for.
+  3. Get approval from at least two existing Maintainers.
```

Comment:
But we defined it should be simple majority (50%) voted, https://github.com/dragonflyoss/community/pull/76.